lib/SIL/AbstractionPattern.cpp

```diff
@@ -256,6 +270,10 @@ const clang::Type *getClangFunctionParameterType(const clang::Type *ty,
 
   // If we're asking about parameters, we'd better have a FunctionProtoType.
   auto fnType = getClangFunctionType(ty)->castAs<clang::FunctionProtoType>();
+  if (index >= fnType->getNumParams()) {
+    fnType->dump();
+    llvm::dbgs() << "Could not get: " << index << "\n";
+  }
```

Comment:
We prefer not to leave this kind of debugging code around.

Reply:
Yes, of course.